app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,16 @@
+class SessionsController < ApplicationController
+
+    def new
+
+    end
+
+    def create
+        user = User.find_by_email(params[:email])
+        if user && user.authenticate(params[:password])
+            session[:user_id] = user.id
+            redirect_to "/"
+        else
+            redirect_to "/sessions/new"
```

Comment:
Will the user see an error message here? Or just find themselves back at the login page with no explanation?

Reply:
Added an error-handling message here, thank you.